plugins/scaffolder/src/components/fields/RepoUrlPicker/RepoUrlPicker.tsx

```diff
@@ -65,10 +79,24 @@ function serializeFormData(data: {
   if (data.organization) {
     params.set('organization', data.organization);
   }
+  if (data.workspace) {
+    params.set('workspace', data.workspace);
+  }
+  if (data.project) {
+    params.set('project', data.project);
+  }
 
   return `${data.host}?${params.toString()}`;
 }
 
+function getIntegrationTypeByHost(host: string, integrations: {
+  type: string,
+  title: string,
+  host: string,
+}[]) {
+  return integrations.filter(integration => integration.host === host)[0].type;
```

Comment:
Regarding my comment on the type above, I see you found how to look up the type of an integration based on the host :grin:. I'm thinking that instead of figuring out the type here and then passing it around as part of the URL, it's probably better to keep the URL slim and just pass the host, then we look up the type again in the backend.